nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplemented
```

Comment:
- [ ] [Optional] You can make your error more specific like so ` raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"`